pyinfra/facts/util/docker.py

```diff
@@ -0,0 +1,20 @@
+import json
```

Comment:
This file appears to be entirely unused?

Reply:
resolved in commit [c961154](https://github.com/pyinfra-dev/pyinfra/pull/1114/commits/c961154d198a89f396ba654a2cadadaf4e424368)